libs/map-editor/src/GameMap/GameMapAreas.ts

```diff
@@ -38,6 +38,13 @@ export class GameMapAreas {
             this.gameMap.tiledObjects
                 .filter((object) => ["zone", "area"].includes(object.class ?? ""))
                 .forEach((areaRaw: ITiledMapObject) => {
+                    const properties = areaRaw.properties ?? [];
+
+                    for (const property of properties) {
+                        if (property.name === "dynamic" && property.type === "bool" && property.value) {
```

Comment:
Sadly we cannot do it like that. Such chance would be like a backdoor for modifying Static Areas via ScriptingAPI without any safety measures.